charts/linkerd2-service-mirror/templates/service-mirror.yaml

```diff
@@ -16,7 +16,7 @@ rules:
   verbs: ["list", "get", "watch", "create", "delete", "update"]
 - apiGroups: [""]
   resources: ["namespaces"]
-  verbs: ["list", "get", "watch"]
+  verbs: ["create","list", "get", "watch"]
```

Comment:
just to clarify, this is unrelated to this change right?  we would need namespace create permission to create mirror services if their namespace doesn't exist but we won't need it to create the gateway mirrors since those will always be in the service-mirror namespace?

Reply:
Yes this is correct